docs/src/main/asciidoc/opentelemetry-metrics.adoc

```diff
@@ -15,7 +15,7 @@ metrics for interactive web applications.
 
 [NOTE]
 ====
-- OpenTelemetry Metrics is disabled by default and Quarkus does not produce automatic metrics for it.
+- OpenTelemetry Metrics is considered _tech preview_ and is disabled by default. Quarkus does not produce automatic metrics for it.
```

Comment:
```suggestion
- OpenTelemetry Metrics is considered _tech preview_ and is disabled by default. 
Quarkus does not produce automatic metrics for it.
```

Reply:
Not quite sure what you mean here by "it". Do you mean "Quarkus does not automatically produce metrics for OTel"?

Reply:
yes, will clarify.

Reply:
Actually that statement is outdated due to the Microprofile Telemetry 2.0 PR that was merged in the meantime. Will just remove the sentence because the Automatic instrumentation section refers to it. This way we can backport with no issues.